discord/appinfo.py

```diff
@@ -213,20 +263,31 @@ def __init__(self, state: ConnectionState, data: AppInfoPayload):
         self.approximate_user_install_count: int | None = data.get(
             "approximate_user_install_count"
         )
-        self.redirect_uris: list[str] | None = data.get("redirect_uris", [])
+        self.approximate_user_authorization_count: int | None = data.get(
+            "approximate_user_authorization_count"
+        )
+        raw_flags = data.get("flags")
+        self._flags: int | None = raw_flags if isinstance(raw_flags, int) else None
+        self.redirect_uris: list[str] = data.get("redirect_uris", [])
         self.interactions_endpoint_url: str | None = data.get(
             "interactions_endpoint_url"
         )
         self.role_connections_verification_url: str | None = data.get(
             "role_connections_verification_url"
         )
+        self.event_webhooks_url: str | None = data.get("event_webhooks_url")
+        self.event_webhooks_status: int | None = data.get("event_webhooks_status")
+        self.event_webhooks_types: list[str] | None = data.get("event_webhooks_types")
 
         install_params = data.get("install_params")
         self.install_params: AppInstallParams | None = (
             AppInstallParams(install_params) if install_params else None
         )
-        self.tags: list[str] | None = data.get("tags", [])
+        self.tags: list[str] = data.get("tags", [])
```

Comment:
Maybe it was None because it could have a value of null? Should it maybe do `or []` to be sure?

Reply:
Oh no I mean removing the None is correct, I meant using `self.tags: list[str] = data.get("tags") or []` to be extra safe.